### PR TITLE
fix-cert-doc

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: docs/content/setup/installation/enterprise_installation.md [hunk 1]
```diff
@@ -160,8 +160,8 @@ echo "creating root cert ..."
 
 openssl req -new -newkey rsa:4096 -x509 -sha256 \
         -days 3650 -nodes -out ${RELAY_ROOT_CERT_NAME}.crt -keyout ${RELAY_ROOT_CERT_NAME}.key \
-        -subj "/CN=*.gloo-mesh/O=${RELAY_ROOT_CERT_NAME}" \
-        -addext "subjectAltName = DNS:*.gloo-mesh"
+        -subj "/CN=enterprise-networking-ca" \
```
Comment (harveyxia) [inline docs/content/setup/installation/enterprise_installation.md:163]: What does this change do?

File: docs/content/setup/installation/enterprise_installation.md [hunk 2]
```diff
@@ -160,8 +160,8 @@ echo "creating root cert ..."
...
+        -addext "extendedKeyUsage = clientAuth, serverAuth"
```
Comment (harveyxia) [inline docs/content/setup/installation/enterprise_installation.md:164]: You had mentioned that you think this is what fixed the user issue, do we know why?

Also do we no longer need `subjectAltName`?

**Comment by EItanya** [comment]
This was all tested locally using `OpenSSL 1.1.1k  25 Mar 2021`

**Comment by joekelley** [comment]
Can you update the notice here with the openssl version you used?

https://github.com/solo-io/gloo-mesh/blob/817bafbc14788b758291c9886c03c62b18bcb58d/docs/content/setup/installation/enterprise_installation.md#L147

**Comment by nmnellis** [comment]
line 253 needs to be updated, it wont let me comment on it!
```    --context ${context} \
```

File: docs/content/setup/installation/enterprise_installation.md [hunk 3]
```diff
@@ -199,15 +199,15 @@ distinguished_name = req_distinguished_name
 [req_distinguished_name]
 [ v3_req ]
 basicConstraints = critical,CA:TRUE
-keyUsage = nonRepudiation, digitalSignature, keyEncipherment, keyCertSign
+keyUsage = digitalSignature, keyEncipherment, keyCertSign
 extendedKeyUsage = clientAuth, serverAuth
```
Comment (nmnellis) [inline docs/content/setup/installation/enterprise_installation.md:203]: When we generate the client certificates for enterprise-agents, do we just set `extendedKeyUsage = clientAuth`?

File: docs/content/setup/installation/enterprise_installation.md [hunk 4]
```diff
@@ -199,15 +199,15 @@ distinguished_name = req_distinguished_name
...
 subjectAltName = @alt_names
 [alt_names]
 DNS = *.gloo-mesh
 EOF
 
 openssl genrsa -out "${RELAY_SIGNING_CERT_NAME}.key" 2048
-openssl req -new -key "${RELAY_SIGNING_CERT_NAME}.key" -out ${RELAY_SIGNING_CERT_NAME}.csr -subj "/CN=*.gloo-mesh/O=${RELAY_SIGNING_CERT_NAME}" -config "${RELAY_SIGNING_CERT_NAME}.conf"
+openssl req -new -key "${RELAY_SIGNING_CERT_NAME}.key" -out ${RELAY_SIGNING_CERT_NAME}.csr -subj "/CN=enterprise-networking-ca" -config "${RELAY_SIGNING_CERT_NAME}.conf"
```
Comment (nmnellis) [inline docs/content/setup/installation/enterprise_installation.md:210]: Does the CN need to be anything specific when generating the client certificates for enterprise-agents?

Comment (nmnellis) [reply docs/content/setup/installation/enterprise_installation.md:210]: Cluster name?

File: docs/content/setup/installation/enterprise_installation.md (hunk 3 above)
Comment (EItanya) [reply docs/content/setup/installation/enterprise_installation.md:203]: Correct

File: docs/content/setup/installation/enterprise_installation.md (hunk 4 above)
Comment (EItanya) [reply docs/content/setup/installation/enterprise_installation.md:210]: No, the cluster name is in the SAN, not the CN

**Review by nmnellis — APPROVED** [review]

**Review by nmnellis — APPROVED** [review]

**Pull request merged**